_content/booklets/recognizing-the-dhamma_geoff.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Recognizing the Dhamma: A study guide"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/Ebooks/RecognizingtheDhamma_181215.pdf"
```

Comment:
Please use https://www.dhammatalks.org/ebook_index.html#RecognizingTheDhamma for the external url